packages/overlays/src/OverlayController.js

```diff
@@ -284,14 +291,30 @@ export class OverlayController {
     const event = new CustomEvent('before-show', { cancelable: true });
     this.dispatchEvent(event);
     if (!event.defaultPrevented) {
-      this._contentNodeWrapper.style.display = this.placementMode === 'local' ? 'inline-block' : '';
+      this._contentNodeWrapper.style.display = this.placementMode === 'local' ? 'flex' : '';
+      this._contentNodeWrapper.style.flexDirection = this.getFlexDirectionAccordingToPosition();
       await this._handleFeatures({ phase: 'show' });
       await this._handlePosition({ phase: 'show' });
       this.elementToFocusAfterHide = elementToFocusAfterHide;
       this.dispatchEvent(new Event('show'));
     }
   }
 
+  getFlexDirectionAccordingToPosition() {
```

Comment:
Can you explain why this is needed?

Reply:
By encapsulating content and arrow inside of the node_wrapper, pepper could not calculate the right position of the tooltip anymore. To fix this I had to change the CSS and use flex instead. so according to the position property the flex-direction is different.

Reply:
Hmm,  Popper 1 or Popper 2? 
Popper 1 seemed to work fine for me...🤔
Maybe I'm wrong ;)

Reply:
I'm using Popper 1. (did not want to upgrade at the same time.) the upgrade can be part of another PR.
In the PR you sent me you upgraded to popper2 so maybe that's why it was working for you.

Reply:
Yeah it should, because it was breaking. When I did the change I first tried on the current version of popper and it seemed to work and then when moving to Popper2 the margin between invoker and content was gone.
Could you maybe elaborate on how those flex-direction props solve the problem?
Does it also work for global overlays?

Reply:
Actually what happens is that when we are in tooltip mode (so local) the arrow itself is not well calculated( not the tooltip). the rationale says that popper wants the arrow to be part of the contentNode. What I did here is that according to the placement you provide I change the flex-direction. So if it's top it's column reverse because the arrow needs to be in the bottom and the tooltip on top. and the opposite for placement: bottom. The same strategy is applied for right and left.

Reply:
Well, if this is the only way to trigger Popper ;) Maybe you could make the method private and make sure it only applies those styles for loval overlays?